### PR TITLE
Adding new supported states to Ios_l3_interfaces resource module

### PR DESCRIPTION
**Pull request by justjais**
##### SUMMARY

Adding new supported states to Ios_l3_interfaces resource module

##### ISSUE TYPE

- Feature Pull Request

##### COMPONENT NAME

ios_l3_interfaces

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

**Comment by justjais** [comment]
recheck

File: plugins/modules/ios_l3_interfaces.py [hunk 1]
```diff
@@ -390,6 +401,143 @@
...
+# After state:
+# ------------
+#
+# vios#sh running-config | section ^interface
+# interface GigabitEthernet0/1
+#  ip address 203.0.113.27 255.255.255.0
+# interface GigabitEthernet0/2
+#  ip address 192.0.2.1 255.255.255.0 secondary
+#  ip address 192.0.2.2 255.255.255.0
+#  ipv6 address 2001:DB8:0:3::/64s
```
Comment (GomathiselviS) [inline plugins/modules/ios_l3_interfaces.py:462]: ipv6 address 2001:DB8:0:3::/64

File: plugins/modules/ios_l3_interfaces.py [hunk 2]
```diff
@@ -93,12 +93,23 @@
...
             - Configures the IPv6 address for Interface.
             type: str
+  running_config:
+    description:
+      - This option is used only with state I(parsed).
+      - The value of this option should be the output received from the VyOS device by executing
+        the command B(show configuration commands | grep interfaces).
+      - The state I(parsed) reads the configuration from C(running_config) option and transforms
+        it into Ansible structured data as per the resource module's argspec and the value is then
+        returned in the I(parsed) key within the result.
```
Comment (GomathiselviS) [inline plugins/modules/ios_l3_interfaces.py:103]: version_added: "1.0.0" to be included.

Comment (justjais) [reply plugins/modules/ios_l3_interfaces.py:103]: @GomathiselviS this I have tc in separate PR: https://github.com/ansible-collections/cisco.ios/pull/35

File: plugins/modules/ios_l3_interfaces.py [hunk 3]
```diff
@@ -39,7 +39,7 @@
 
 
 DOCUMENTATION = """module: ios_l3_interfaces
-short_description: Manage Layer-3 interface on Cisco IOS devices.
+short_description: Layer-3 interface resource module
```
Comment (rohitthakur2590) [inline plugins/modules/ios_l3_interfaces.py:42]: would it be "L3 interfaces resource module" ?

**Comment by justjais** [comment]
recheck

**Comment by justjais** [comment]
recheck

**Comment by justjais** [comment]
recheck

**Pull request merged**